doc/classes/NetworkedMultiplayerCustom.xml

```diff
@@ -6,6 +6,66 @@
 	<description>
 		A [NetworkedMultiplayerPeer] implementation that can be used as a [member MultiplayerAPI.network_peer] and controlled from a script.
 		Its purpose is to allow adding a new backend for the high-Level multiplayer API without needing to use GDNative.
+		Here's a short example, assuming you have an imaginary [code]_connect_to_real_multiplayer()[/code] function that will create a client connected to the underlying multiplayer transport (ex. Steam peer-to-peer, PlayFab, etc):
```

Comment:
This only illustrates joining and seems like it should also include how a host should set up the custom peer.

Reply:
If you'd like, I wouldn't mind writing up the C# equivalent for this as well when I get a minute.

Reply:
> This only illustrates joining and seems like it should also include how a host should set up the custom peer.

Hm. The only things that would be different for a host, is that you might use a different imaginary method, and you'd set the peer_id to 1. I don't know that it's worth complicating things with more imaginary methods, but I'll add a comment with a note about setting the peer_id to 1.

> If you'd like, I wouldn't mind writing up the C# equivalent for this as well when I get a minute.

That would be awesome! Thanks :-)

Reply:
> Hm. The only things that would be different for a host, is that you might use a different imaginary method, and you'd set the peer_id to 1. I don't know that it's worth complicating things with more imaginary methods, but I'll add a comment with a note about setting the peer_id to 1.
Yes a note about that would be good and then I think it's sufficient.
This the way I actually implemented the initialize line `client.Initialize(isHost ? 1 : (int)realPeerId);` but yeah the rest of the init method was the same.

Reply:
I just pushed an update!